grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -260,4 +260,13 @@ class CycleCountService {
 
         return availableItems
     }
+
+    List<CycleCountDto> getCycleCounts(List<String> ids) {
+        List<CycleCount> cycleCounts = CycleCount.createCriteria().list {
```

Comment:
I think you can do `CycleCount.findByIdIn(ids)`

Reply:
I wanted to allow fetching all if you don't provide the ids, so this wouldn't work with version you proposed. I think those versions are anyway almost equal, it's just "syntatic sugar".